src/material/tabs/testing/tab-harness-filters.ts

```diff
@@ -7,14 +7,23 @@
  */
 import {BaseHarnessFilters} from '@angular/cdk/testing';
 
-/** A set of criteria that can be used to filter a list of `MatRadioButtonHarness` instances. */
+/** A set of criteria that can be used to filter a list of `MatTabHarness` instances. */
 export interface TabHarnessFilters extends BaseHarnessFilters {
   /** Only find instances whose label matches the given value. */
   label?: string | RegExp;
 }
 
-/** A set of criteria that can be used to filter a list of `MatRadioButtonHarness` instances. */
+/** A set of criteria that can be used to filter a list of `MatTabGroupHarness` instances. */
 export interface TabGroupHarnessFilters extends BaseHarnessFilters {
   /** Only find instances whose selected tab label matches the given value. */
   selectedTabLabel?: string | RegExp;
 }
+
+/** A set of criteria that can be used to filter a list of `MatTabLinkHarness` instances. */
+export interface TabLinkHarnessFilters extends BaseHarnessFilters {
+  /** Only find instances whose label matches the given value. */
+  label?: string | RegExp;
```

Comment:
Would it make sense to have a filter for `href` as well?

Reply:
I think that it makes sense, but the `mat-tab-link` isn't guaranteed to be an anchor tag since it's using an attribute selector.